{PROJECT_NAME}/Configurations/XCConfigs/DebugStaging.xcconfig

```diff
@@ -7,7 +7,8 @@ ONLY_ACTIVE_ARCH = YES
 SWIFT_OPTIMIZATION_LEVEL = -Onone
 
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) DEBUG=1 STAGING=1
-PRODUCT_NAME = $(TARGET_NAME) Staging
+APP_DISPLAY_NAME = $(TARGET_NAME) Staging
+PRODUCT_NAME = $(TARGET_NAME)
```

Comment:
We can safely remove all the `PRODUCT_NAME` entries in all `xcconfig` files.

It's because its value is always set to `$(TARGET_NAME)` by default. Also, whatever value is set here actually has no effect since, in the end, Xcode will resolve its value with the one provided at the target-level 🙏 (Meaning if we want to override, we have to override at the target-level, not at the config file level).